README.md

```diff
@@ -1 +1,5 @@
-# GitHubBot
\ No newline at end of file
+# GitHub Bot
```

Comment:
This README does not describe the GitHub PR Bot well. Please note that the GitHub PR Bot is a WEB API which uses GitHub webhooks for PR Review Comments, when a new review comment is left, it reads the comment, analyzes the code in question and then comments a suggested code solution.